site/app/templates/forum/CreatePost.twig

```diff
@@ -1,28 +1,28 @@
 <div class="{% for class in classes %} {{ class }} {% endfor %}" id="{{ post_id }}" style="margin-left:{{ offset }}px;" reply-level="{{ reply_level }}">
     {% if first == true %}
-        <h3 style="max-width: 95%; display:inline-block;word-wrap: break-word;margin-top:10px; margin-left: 5px;">
+        <h3 class="create-post-head">
 
             {% if userGroup <= 2 and activeThreadAnnouncement %}
 
-                <a style="display:inline-block; color:orange; " onClick="alterAnnouncement({{ activeThread.id }}, 'Are you sure you want to remove this thread as an announcement?', 'remove_announcement', '{{ csrf_token }}')" title="Remove Announcement"><i class="fas fa-star" onmouseleave="changeColor(this, 'gold')" onmouseover="changeColor(this, '#e0e0e0')" style="position:relative; display:inline-block; color:gold; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" aria-hidden="true"></i></a>
+                <a class="active-thread-announcement-1" onClick="alterAnnouncement({{ activeThread.id }}, 'Are you sure you want to remove this thread as an announcement?', 'remove_announcement', '{{ csrf_token }}')" title="Remove Announcement"><i class="fas fa-star" onmouseleave="changeColor(this, 'gold')" onmouseover="changeColor(this, '#e0e0e0')" aria-hidden="true"></i></a>
 
             {% elseif activeThreadAnnouncement %}
 
-                <i class="fas fa-star" style="position:relative; display:inline-block; color:gold; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title = "Announcement" aria-hidden="true"></i>
+                <i class="fas fa-star active-thread-announcement-2" title = "Announcement" aria-hidden="true"></i>
```

Comment:
Can we have more descriptive names here for active-thread-announcement-1/2? I think it makes more sense to have a descriptor rather than a number identifier.

Reply:
Done!